pkg/volume/parameter.go

```diff
@@ -76,7 +76,7 @@ type Parameters struct {
 	// Encrypt volumes if true.
 	Encryption bool
 	// AllowRemoteVolumeAccess if true, volumes may be accessed over the network.
-	AllowRemoteVolumeAccess bool
```

Comment:
so that is a breaking change or did I overlook something?

Reply:
Not really. `true` and `false` are now syntax sugar (see commit message). Observable behaviour should remain the same

Reply:
right, that was me overlooking the `MarshalText/UnmarshalText`